### DISCUSSION TITLE
Gogardian Password

### DISCUSSION THREAD

**Discussion by Wynfhi** [3kh0/ext-remover#391] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My friend found a way to get a lot of admin credentials. Does anyone know a way to do this or is he just special.

**Comment by ghost** [comment, marked as answer]
My friend can run in a certain way. I'm not going to specify which, but I expect you to guess how. Does anybody else have the incredible guessing powers to infer the meaning of what I'm trying to say?

**Comment by Alex2Mayhem** [comment]
He's lying. But your also special.
You're basically saying your dad went to the store to get the milk and came back.  But you don't know how instead of asking your dad who came back with the milk you go somewhere else thinking they know.

**Reply by Alex2Mayhem** [reply]
Boom there is your Anwser.

**Reply by Aka-but-nice** [reply]
> He's lying. But your also special. You're basically saying your dad went to the store to get the milk and came back. But you don't know how instead of asking your dad who came back with the milk you go somewhere else thinking they know.

I really didn't understand tf you said, jit' speakin' shadow realm frfr
